lib/pages/my_reports_pages/my_reports_page.dart

```diff
@@ -579,206 +574,204 @@ class _MyReportsPageState extends State<MyReportsPage> {
                           ),
                         ],
                       ),
-                      isMine
```

Comment:
in the lines below I didn't change logic, just remove "isMine" which makes it seem like many lines were changed